src/App.jsx

```diff
@@ -15,7 +16,8 @@ export function App() {
         <Route path="" element={<Layout withSidebar />}>
           <Route index element={<Navigate to="dashboard" />} />
           <Route path="dashboard/*" element={<Dashboard />} />
-          <Route path="exercises/*" element={<Exercises />} />
+          <Route path="my-cv/*" element={<Dashboard />} />
+          <Route path="exercises/*" element={<MyCV />} />
```

Comment:
Tu widzę małego chochlika 😄 dla route'a `my-cv` masz ustawiony komponent `Dashboard`, a dla `exercises` masz `MyCV`, ale to pewnie od razu zauważysz przy klikaniu, że coś jest nie teges 😃

Reply:
To prawda, naprawilam to na nastepnych zajeciach bo cos nie pykło :)